src/main/java/uk/ac/ebi/eva/pipeline/configuration/writers/VepInputFlatFileWriterConfiguration.java

```diff
@@ -30,8 +30,8 @@
 
     @Bean(VEP_INPUT_WRITER)
     @StepScope
-    public ItemStreamWriter<VariantWrapper> vepInputFlatFileWriter(JobOptions jobOptions) {
-        return new VepInputFlatFileWriter(jobOptions.getVepInput());
+    public ItemStreamWriter<VariantWrapper> vepInputFlatFileWriter(AnnotationParameters annotationParameters) {
+        return new VepInputFlatFileWriter(annotationParameters.getVepInput());
```

Comment:
Please discuss this with @jmmut , we agreed yesterday on placing path resolution (for statistics) in the `URLHelper` class.